monkey/tests/unit_tests/monkey_island/cc/environment/test_windows_permissions.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+import pytest
+
+from monkey_island.cc.environment.windows_permissions import set_perms_to_owner_only
+
+
+@pytest.mark.skipif(os.name == "posix", reason="Tests Windows (not Posix) permissions.")
+def test_set_perms_to_owner_only(tmpdir):
```

Comment:
We should run this test against `create_secure_directory()`. This saves you some setup and should continue to pass after we fix #1193